src/components/Follow/FollowListUsers.tsx

```diff
@@ -35,12 +36,16 @@ export default function FollowListUsers({
     track('Follower List Username Click');
   }, [track]);
 
+  const trimBreaklines = useCallback((text: string) => {
+    return text.replace(BREAK_LINES, '');
+  }, []);
```

Comment:
we should just be able to useMemo here!

```js
const bio = useMemo(() => user.bio.replace(BREAK_LINES, ''), [user.bio])
```

Reply:
fml. why did I used `useCallback` here 🤦🏻‍♂️

Reply:
Updated!